tests/api/test_account.py

```diff
@@ -15,6 +15,39 @@
 
 from .utils import assert_no_permission, convert_dict_keys_to_camel_case
 
+UPDATE_LOGGED_CUSTOMER_QUERY = """
+    mutation UpdateLoggedCustomer($billing: AddressInput,
+                                  $shipping: AddressInput) {
+        loggedUserUpdate(
+          input: {
+            defaultBillingAddress: $billing,
+            defaultShippingAddress: $shipping,
+        }) {
+            errors {
+                field
+                message
+            }
+            user {
+                email
+                defaultBillingAddress {
+                    id
+                }
+                defaultShippingAddress {
+                    id
+                }
+            }
+        }
+    }
+"""
+USER_EMAIL_QUERY = """
```

Comment:
I like the idea of keeping that query just above the tests using it, so going back and forward is way easier.